### PR TITLE
[PKG-2616] sphinx-design 0.5.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/executablebooks/sphinx-design/blob/v0.5.0/CHANGELOG.md
License: https://github.com/executablebooks/sphinx-design/blob/v0.5.0/LICENSE
Requirements: https://github.com/executablebooks/sphinx-design/blob/v0.5.0/pyproject.toml

Actions:
1. Skip py<37 and remove noarch python
2. Fix pinnings in host and run
3. Skip `test_sd_hide_title_myst` and `test_sd_hide_title_rst` as the packages myst-parser and pytest-regressions package isn't available on the defaults yet
4. Add dev & doc urls
5. Add a description

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,49 +1,55 @@
...
+    #- myst-parser
+    #- pytest-regressions
   source_files:
     - tests/
-  imports:
-    - sphinx_design
   commands:
-    - pytest tests -vv
+    - pip check
+    # Skip `test_sd_hide_title_myst` and `test_sd_hide_title_rst` as the packages myst-parser and pytest-regressions package isn't available on the defaults yet
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:40]: do not we want to deliver those dependencies along with this package ?

Comment (skupr-anaconda) [reply recipe/meta.yaml:40]: No, they are required for testing only https://github.com/executablebooks/sphinx-design/blob/18fe5dfb025204f0d797ddde3d220e9682e9db70/pyproject.toml#L39-L43

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by ryanskeith — APPROVED** [review]
LGTM

**Pull request merged**